### PR TITLE
Expanded breadcrumb.

### PR DESCRIPTION
**Pull request by kantrowitz**
Addresses #1694 Expanded breadcrumb.

### CODE REVIEW

**Comment by awong-dev** [comment]
@webinista I pulled down this change. Something seems borked with the plugin code. It seems like maybe an off-by-one error?

**Comment by webinista** [comment]
@awong-dev No. The plugin weird and not entirely intuitive, but it works.  

Here's what's going on: the third/last breadcrumb will _always_ match the `title` attribute that's in the corresponding Markdown file. You actually _can't_ override it, despite lines in `_config.yml` that suggest that you can. (Which is why I gave @kantrowitz bad info. Sorry about that.) 

As it turns out, we actually just need to change the title of `_employment/job-seekers/veci.md` from "VECI" to "Veterans Economic Community Initiative." But doing so will cause the breadcrumbs to wrap. I suspect that's why we changed the title to VECI in the first place.

screen shot 2016-04-06 at 2 46 54 pm

**Comment by emilyville** [comment]
Breadcrumb wrapping is ok - this already happens a bunch. It's not ideal but this won't be the first!

**Comment by webinista** [comment]
Ok. In that case, this PR is ready for a review.

**Comment by kantrowitz** [comment]
Should we change this in the readme, then?

File: _config.yml [hunk 1]
```diff
@@ -84,16 +84,13 @@ collections:
...
       type: "disability-benefits"
     values:
       layout: "page-breadcrumbs"
       body_class: "page-disability"
   - scope:
-      path: "disability-benefits/conditions/exposure-to-hazardous-materials"
-    values:
-      breadcrumb_3: "Exposure to Hazardous Materials"
-  - scope:
       path: "disability-benefits/apply-for-benefits/"
```
Comment (emilyville) [inline _config.yml]: Why did we remove this here? Not sure if this was intentional - or just an instance of V2 being out of sync with master

File: _config.yml [hunk 2]
```diff
@@ -108,10 +105,6 @@ defaults:
     values:
       breadcrumb_2: "GI Bill"
   - scope:
-      path: "education/gi-bill/survivors-dependent-assistance"
-    values:
-      breadcrumb_3: "Survivors' and Dependents' Assistance"
-  - scope:
```
Comment (emilyville) [inline _config.yml]: Same here

File: _config.yml [hunk 3]
```diff
@@ -128,10 +121,6 @@ defaults:
     values:
       breadcrumb_2: "Apply for Education Benefits"
   - scope:
-      path: "education/work-learn/non-traditional"
-    values:
-      breadcrumb_3: "Non-Traditional Options"
-  - scope:
```
Comment (emilyville) [inline _config.yml]: and here!

**Comment by emilyville** [comment]
I have a hunch the only things meant to change in this PR are the readme and the _employment/job-seekers/veci.md title?

If so, there's an extra bunch of changes in the config file that seem like they snuck in. Back at you guys, @kantrowitz and @webinista

**Comment by webinista** [comment]
No. That's correct. I removed those lines from `_config.yml` because they're misleading. The Markdown file titles are what's defining the text in those breadcrumbs, not the `breadcrumb_3` value.

**Comment by emilyville** [comment]
great! I should have read the readme more thoroughly. :D Thanks @webinista and I will go ahead and merge - LGTM.

**Pull request merged**